src/components/templates/ExternalRoom/ExternalRoom.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { useEffect } from "react";
+
+export interface ExternalRoomProps {
+  zoomUrl: string;
+}
+export const ExternalRoom: React.FC<ExternalRoomProps> = ({ zoomUrl }) => {
+  useEffect(() => {
+    window.open(zoomUrl);
+  }, [zoomUrl]);
```

Comment:
reuse utils functions to open external source

Reply:
fixed in 03fcfc07bda6c3d8df4d15a25ea8f36bb1956118